experiment/coverage/apicoverage.go

```diff
@@ -137,7 +165,16 @@ func getAPILog(restlog string) apiArray {
 	}
 	defer fp.Close()
 
-	return parseAPILog(fp)
+	apisLog := parseE2eAPILog(fp)
+	if len(apisLog) > 0 {
+		return apisLog
+	}
+	//NOTE: If the log is not taken from e2e side, here tries to parse it as apiserver side log.
```

Comment:
I did think this kind of flag option on the command, but I implemented this way because I was lazy.

Anyways it is fine to add some flag to choose which type of log is specified to make user understand clearly :-)